src/deploy/functions/containerCleaner.ts

```diff
@@ -10,10 +10,11 @@ import { logger } from "../../logger";
 import * as docker from "../../gcp/docker";
 import * as backend from "./backend";
 import * as utils from "../../utils";
+import { FirebaseError } from "../../error";
 
 // A flattening of container_registry_hosts and
 // region_multiregion_map from regionconfig.borg
-const SUBDOMAIN_MAPPING: Record<string, string> = {
+export const SUBDOMAIN_MAPPING: Record<string, string> = {
```

Comment:
I added an export here so I can use the actual object in unit testing for deleting every region. I didn't want to create a 2nd source of truth and have failures if this ever gets updated